src/Uno.UI.Runtime.Skia.Wpf/Controls/WpfTextViewTextBox.cs

```diff
@@ -1,16 +1,15 @@
 ﻿using System.Windows.Controls;
 using WpfFrameworkPropertyMetadata = System.Windows.FrameworkPropertyMetadata;
 
-namespace Uno.UI.Runtime.Skia.WPF.Controls
+namespace Uno.UI.Runtime.Skia.Wpf.Controls;
```

Comment:
The namespce change here is a breaking change. Do we want to take it for `master`?

Reply:
It is, but it is also very unlikely the control is used by someone outside of Uno directly. Actually, the whole class should be internal instead. 🤔